libraries/radwordsprocessing/formats-and-conversion/rtf/rtfformatprovider.md

```diff
@@ -67,6 +73,9 @@ __Example 3__ shows how to use __RtfFormatProvider__ to export __RadFlowDocument
 #### __[C#] Example 3: Export a document to a file__
 
 {{region cs-radwordsprocessing-formats-and-conversion-rtf-rtfformatprovider_2}}
+	using Telerik.Windows.Documents.Flow.FormatProviders.Rtf;
+	using Telerik.Windows.Documents.Flow.Model;
```

Comment:
Do not add usings to the snippet - it won't build when directly pasted. Instead, use full class names.